vulpes/memory/gamestate/table.hpp

```diff
@@ -18,19 +20,23 @@ class Table {
     // Returns true if succesful, false if it couldn't find it.
     bool remove(MemRef id);
 
+    size_t count();
+
     char name[0x20];
-    int16_t max_count;
-    int16_t index_size;
-    bool valid;
+    int16_t max_elements;
+    int16_t element_size;
+    bool is_valid;
     bool id_zero_valid;
     PAD(2);
-    int32_t sig; // d@t@
-    int16_t next_id;
-    int16_t last_id;
-    MemRef next;
+    int32_t sig; // "d@t@"
+    int16_t next_id; // Next index to use when creating a new element
+    int16_t last_id; // Last used id
+    MemRef next_data; // Next id value to use
 }; static_assert(sizeof(Table) == 0x34);
```

Comment:
I can puzzle out where 0x34 comes from, but it should be a named constant to avoid that unnecessary mental processing power.

Reply:
These struct sizes are only ever used in pairing with these static asserts that we only put at the end of structs. Is there any advantage to defining them separately at that point?